File: src/main/java/mods/eln/sim/SignalRp.java
package mods.eln.sim;

import mods.eln.Eln;
import mods.eln.sim.mna.component.Resistor;
import mods.eln.sim.mna.state.State;

public class SignalRp extends Resistor {
    public SignalRp(State aPin) {
        super(aPin, null);
        setResistance(Eln.SVU / Eln.SVII);
    }
}

File: src/main/java/mods/eln/sim/SignalRp.kt
package mods.eln.sim

import mods.eln.Eln
import mods.eln.sim.mna.component.Resistor
import mods.eln.sim.mna.state.State

class SignalRp(aPin: State) : Resistor(aPin, null) {
    init {
        r = Eln.SVU / Eln.SVII
    }
}
